unittest/test_load.py

```diff
@@ -57,6 +57,18 @@ def test_anymal_kinova(self):
     def test_baxter(self):
         self.check("baxter", 19, 19)
 
+    def test_bravo7_gripper(self):
+        self.check("bravo7_gripper", 11, 8)
+
+    def test_bravo7_sphere_endeffector(self):
+        self.check("bravo7_sphere_endeffector", 9, 6)
+
+    def test_falcon_bravo7_sphere_endeffector(self):
+        self.check("falcon_bravo7_sphere_endeffector", 16, 12)
+
+    def test_bluevolta_bravo7_sphere_endeffector(self):
+        self.check("bluevolta_bravo7_sphere_endeffector", 16, 12)
```

Comment:
Update this
```suggestion
    def test_bravo7_no_ee(self):
        self.check("bravo7_no_ee", 9, 6)

    def test_falcon_bravo7_no_ee(self):
        self.check("falcon_bravo7_no_ee", 16, 12)

    def test_bluevolta_bravo7_no_ee(self):
        self.check("bluevolta_bravo7_no_ee", 16, 12)
```

Reply:
Renaming and descriptions for Bluevolta+Bravo7+gripper and Falcon+Bravo7+gripper have been updated.